os/board/rtl8720e/src/libs/.gitignore

```diff
@@ -1,5 +0,0 @@
-!./lib_enc.a
-!./lib_enc_fpu.a
-!./lib_wps.a
-!./lib_wps_fpu.a
-!./cmse_implib.a
```

Comment:
They have `!` so that original is correct to keep .a in the git.
I think we should revert this file.

Reply:
Sorry, could you explain this comment please.

Reply:
`!` means NOT. Because the .gitignore at the root has `*.a`, we can't push .a file in the git. So, this `!` allows to merged even it is .a file.